build.fsx

```diff
@@ -171,6 +171,28 @@ Target "ReleaseGitHub" (fun _ ->
     |> Async.RunSynchronously
 )
 
+// .NET Core SDK and .NET Core
+
+let assertExitCodeZero x = if x = 0 then () else failwithf "Command failed with exit code %i" x
+
+Target "Build.NetCore" (fun _ ->
+    Shell.Exec("dotnet", "restore") |> assertExitCodeZero
+    Shell.Exec("dotnet", "--verbose pack --configuration Release", "src/Argu") |> assertExitCodeZero
+)
+
+Target "RunTests.NetCore" (fun _ ->
+    Shell.Exec("dotnet", "--verbose run --configuration Release", "tests/Argu.NetCore.Tests") |> assertExitCodeZero
+)
+
+let isDotnetSDKInstalled = try Shell.Exec("dotnet", "--version") = 0 with _ -> false
+
+Target "Nuget.AddNetCore" (fun _ ->

```

Comment:
@forki can we use argu or better chessie as the example for fake new dnc helpers? 

In another pr if possibile, so we can iterate the design. 

Atm this workflow work really well, checklist of steps, i'd like to make people use that atm, and update to fake helpers later. 
i think that can speedup a little .net core adoption.

Also i think current one is not too bad, because make easy to see dotnet command, so can be run locally, no magic (atm people see dnc  sdk as magic)